blimflix/client/app/directives/movie-filters/movie-filters.directive.html

```diff
@@ -0,0 +1,28 @@
+<div class="container-filters">
+  <p class="heading">{{title}} ({{qty}})</p>
+  <form class="navbar-form" role="search">
+    <div class="input-group add-on">
+      <input class="form-control" placeholder="Search" type="text" ng-model="name">
+      <div class="input-group-btn">
+        <button ng-click='search(name)' class="btn btn-default lupa" type="submit"><i class="glyphicon glyphicon-search"></i></button>
+      </div>
+    </div>
+  </form>
+  <select
+    class="form-control"
+    ng-change="filterMovies()"
+    ng-model="genre"
+    ng-options="value for value in genres">
+    <option value="">Select genre</option>
+    <option></option>
+  </select>
+  <select
+    class="form-control form-year"
+    ng-change="filterMovies()"
+    ng-model="year"
+    ng-options="value for value in years">
+    <option value="">Select year</option>
+    <option></option>
+  </select>
+  <img class="poster" ng-repeat="x in filtered" src={{x.poster}} alt="Smiley face" height="150" width="100">
```

Comment:
please use ng-src instead of src
the reason is explained in the link
http://stackoverflow.com/questions/26833500/angular-img-src-console-error/#answers-header

the first request for image will be http://{{x.poster}}